### DISCUSSION TITLE
Path loss model RT sionna

### DISCUSSION THREAD

**Discussion by Virusx51** [NVlabs/sionna#288] [Q&A] [answered]
Hi, friends
What is the signal attenuation model in Sionna ray tracing? How do the coefficients of the impulse response depend on distance? Please provide references to the literature.

**Comment by jhoydis** [comment, marked as answer]
Hi, please refer to the [EM Primer](https://nvlabs.github.io/sionna/em_primer.html) which gives a very detailed explanation of how Sionna RT is implemented. You might also want to refer to this [paper](https://arxiv.org/abs/2311.18558).

**Comment by Virusx51** [comment]
Thank you very much, I will definitely look into it.
